src/switcher-ui.tsx

```
@@ -528,83 +588,74 @@ function SwitcherApp() {
+      // Identical rows in identical order → keep the old array. Some of the
+      // deps below are refreshed by polling, and a fresh array every tick
+      // would re-render the list for nothing.
+      if (
+        next.length === prev.length &&
+        next.every((s: any, i: number) => s.sessionId === prev[i].sessionId)
+      ) {
+        return prev;
+      }
+      return next;
+    });
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

ESLint

[error] Definition for rule 'react-hooks/exhaustive-deps' was not found.

(react-hooks/exhaustive-deps)